activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -107,7 +107,8 @@ def take!
     end
 
     # Find the first record (or first N records if a parameter is supplied).
-    # If no order is defined it will order by primary key.
+    # If no order is defined it will order by primary key if the collection is not loaded.
+    # If the collection is loaded it will get the first record from that collection.
```

Comment:
How about

```
# If the records are not already present in memory, it will order by primary key if no order is specified.
# When the records are already in memory it will return the first record.
```

Instead of collection, we can say `records` so we drop reference to `the collection` and `loaded`.
 
@sgrif Does that sound good to you?